maven-surefire-common/src/main/java/org/apache/maven/surefire/providerapi/ConfigurableProviderInfo.java

```diff
@@ -1,4 +1,4 @@
-package org.apache.maven.plugin.surefire;
+package org.apache.maven.surefire.providerapi;
```

Comment:
@slawekjaranowski Why the package has changed?

Reply:
I put all classes responsible for providers detecting in one package.

Before we have:
`org.apache.maven.plugin.surefire.booterclient.ProviderDetector` 
which use 
`org.apache.maven.surefire.providerapi.ServiceLoader` 
and `ProviderDetector` was used only by `AbstractSurefireMojo` so one responsibility was in many package.

Reply:
yes, I see there are now 5 classes. LGTM